webapp/components/Comment/Comment.vue

```diff
@@ -168,6 +176,23 @@ export default {
         this.$toast.error(err.message)
       }
     },
+    answerComment() {
+      const slug =
+        '<a class="mention" href="/profile/' +
+        this.comment.author.id +
+        '" data-mention-id="' +
+        this.comment.author.slug +
+        '" target="_blank" contenteditable="false">@d' +
+        this.comment.author.slug +
+        '</a>'
+      document.querySelector('.editor-content div').focus()
```

Comment:
Don't use globals like `document` which are not available in a NodeJS environment.

Reply:
wir haben vue-scroll over nuxt in the system 
you can ever on the element set: 
` v-scroll-to="'.editor'"`

Reply:
Scrolling != focus

You can scroll to an element programatically: https://github.com/rigor789/vue-scrollto#programmatically with `this.$scrollTo(element, duration, options)`.